src/full-screen/components/runtime.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import screenfull from "screenfull";
+import { IAuthoredState } from "./types";
+import { FullScreenButton } from "./full-screen-button";
+import css from "./runtime.scss";
+
+interface IProps {
+  authoredState: IAuthoredState;
+}
+
+export const Runtime: React.FC<IProps> = ({ authoredState }) => {
+  const [ isFullScreen, setIsFullScreen ] = useState<boolean>(false);
+
+  const onToggleFullScreen = () => {
+    if (screenfull.isEnabled) {
+      screenfull.toggle();
+      screenfull.on("change", () => {
```

Comment:
We shouldn't be adding this handler on every toggle. Instead, it should be added in a `useEffect` which can then remove the handler in its cleanup function.